### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by lainz** [bgrabitmap/bgracontrols#182]
Hi, there are more controls and more authors now, is needed to update the readme file with the new stuff.

**Comment by BobanSpasic** [comment]
With or without new screenshots?

Apart from BCFluent and BCLea components, is there anything else to add?

**Comment by BobanSpasic** [comment]
I've added some changes in the dev branch.

**Comment by lainz** [comment]
Hi. As you wish if you want you can add screenshots.

**Comment by lainz** [comment]
Hi please add maxm74 as author of bgraimagemanipulation.

**Comment by BobanSpasic** [comment]
I'll leave it with just the added text for now.
An extra showcase would be a better thing to have, with apps using BGRA Controls and/or a showcase of the controls.

**Comment by lainz** [comment]
Yes that will be nice to have. For example we can add a lazpaint screenshot.

And your theme editor screenshot.

**Comment by BobanSpasic** [comment]
> Hi please add maxm74 as author of bgraimagemanipulation.

Done

**Comment by lainz** [comment]
Thanks.

I'm sleeping here is 7am in Argentina and I usually wake up at 8:45am.

I take some medicine always at 6am and today I can't back to sleep.

So all mails are welcome.

**Comment by BobanSpasic** [comment]
> Yes that will be nice to have. For example we can add a lazpaint screenshot.
> 
> And your theme editor screenshot.

I'll open a topic on Lazarus forum, to ask people about contributing screenshots of their apps.

**Comment by lainz** [comment]
Cool. I will share one of my free apps.

The others I can't because are property of the company where I work.

**Comment by BobanSpasic** [comment]
https://forum.lazarus.freepascal.org/index.php/topic,67716.0.html

**Comment by lainz** [comment]
Thanks. I've shared my app there.

**Comment by maxm74** [comment]
> Hi please add maxm74 as author of bgraimagemanipulation.

Thanks to all of you.
